### DISCUSSION TITLE
Add a pocketbase record using DevTools console

### DISCUSSION THREAD

**Discussion by Joedotmt** [pocketbase/pocketbase#4205] [Q&A]
Hello, I have a very specific need.
I would like to add a PocketBase record using DevTools console.
Below you may find the way to add a record normally using the JS api
```JS
import PocketBase from 'pocketbase';
const pb = new PocketBase('http://127.0.0.1:8090');
const data = {
    "isbn": "3049850954834",
    "image": "https://www.example.com/image.jpg"
};
const record = await pb.collection('book_covers').create(data);
```

**Comment by jonshipman** [comment]
Create a file called `pb_hooks/book-covers.cmd.pb.js` and add the following:

```
$app.rootCmd.addCommand(
  new Command({
    use: "book_covers",
    run(cmd, args) {
      if (!args[0]) return console.error("Need isbn");
      if (!args[1]) return console.error("Need image");

      const [isbn, image] = args;

      $app.dao().db().insert("book_covers", { isbn, image }).execute();
    },
  })
);
```

Now you can run the command `./pocketbase book_covers 3049850954834 https://www.example.com/image.jpg`

**Reply by Joedotmt** [reply]
It is not working for me
<img width="463" alt="image" src="https://github.com/pocketbase/pocketbase/assets/99688750/5606512c-ea9d-49f1-811d-a007bd123e5a">

**Reply by jonshipman** [reply]
You need to quote the url, and learn how to debug.
